src/python/pants/engine/internals/graph.py

```diff
@@ -227,15 +227,23 @@ async def resolve_target_parametrizations(
                 if field_value is not None:
                     template_fields[field_type.alias] = field_value
 
+        field_type_aliases = target_type._get_field_aliases_to_field_types(
```

Comment:
@stuhood the two `for` loops here on lines 221 and 225 doesn't seem to address the case of deprecated field aliases..?

Reply:
No trouble, simply observing :)